src/platform/src/StructuredOutput/PlatformSubscriber.php

```diff
@@ -36,6 +36,7 @@
  */
 final class PlatformSubscriber implements EventSubscriberInterface
 {
+    public const KEY_OUTPUT_STRUCTURE = 'response_format';
```

Comment:
that's a bit tricky here - we have that `output_structure` key for using it with `Some\Class::`, but internally we convert that to `response_format`, that is than picked up by the model clients and potentially converted one step further to something model/endpoint specific.

what do you think about streamlining to a single key (maybe `output`) that just checks if `class_exists($options['output'])` and otherwise assumes that it is an explicit structure description like in the `structured-output-clock.php` example?

i def like having a constant for that tho :+1: